### PR TITLE
geojson fixed to set leafletData object once, and to use cloned model on add

### PR DESCRIPTION
**Pull request by nmccready**

### CODE REVIEW

File: test/unit/leafletData/leafletDataSpec.coffee [hunk 1]
```diff
@@ -0,0 +1,69 @@
+describe 'leafletData', ->
```
Comment (nmccready) [inline test/unit/leafletData/leafletDataSpec.coffee]: double checking leafletData logic

File: grunt/karma.js [hunk 2]
```diff
@@ -46,11 +46,14 @@ module.exports = function (grunt, options) {
             //logLevel: 'DEBUG',
             reporters: ['progress', 'coverage'],
             preprocessors: {
+                'test/**/**/*.coffee': ['coffee'],
```
Comment (nmccready) [inline grunt/karma.js]: makes coverage work again

**Comment by nmccready** [comment]
@zacronos @tombatossals

**Comment by nmccready** [comment]
@jessertaylor

**Comment by tombatossals** [comment]
Fantastic!

**Comment by jessertaylor** [comment]
excellent will give it a go

**Pull request merged**